docker-compose.yml

```
@@ -0,0 +1,8 @@
+version: "3.7"
+
+services:
+    api:
+        build:
+            context: .
+        ports:
+            - '3000:${PORT}'
```

```suggestion
            - "3000:${PORT:-3000}"
```
Put a default value for environment variable.